rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -0,0 +1,245 @@
+import logging
+from typing import Dict, Text, List, Any
+
+import rasa.utils.io
+from rasa.constants import DOCS_URL_STORIES
+from rasa.core.constants import INTENT_MESSAGE_PREFIX
+from rasa.core.events import UserUttered, SlotSet
+from rasa.core.training.story_reader.story_reader import StoryReader
+from rasa.core.training.structures import StoryStep
+from rasa.data import YAML_FILE_EXTENSIONS
+from rasa.utils.common import raise_warning
+
+logger = logging.getLogger(__name__)
+
+KEY_STORIES = "stories"
+KEY_PLAIN_STORY_NAME = "story"
+KEY_PLAIN_STORY_STEPS = "steps"
+KEY_PLAIN_STORY_USER_UTTERANCE = "user"
```

Comment:
what's a "plain user utterance"?

Reply:
It's a "plain story", as opposed to the "e2e story" in the future.

Reply:
mhm, not sure if we need to distinguish.  e2e stories gonna be just regular stories which are hybrids (so intent/ action labels mixed with actual user / bot messages.

Reply:
don't we want to define them with a different YAML key?
Anyways, could you please add an example of the e2e story to the 2.0 Yaml doc?

Reply:
> don't we want to define them with a different YAML key?

No. As said, they are gonna be hybrid.

> Anyways, could you please add an example of the e2e story to the 2.0 Yaml doc?

I thought we are still discussing this in the Slack thread with Vova tbh since we not came to a solution

Reply:
oh ok then.